shared/scss/base/_reset.scss

```diff
@@ -1,3 +1,7 @@
+li {
+  list-style-type: none;
+}
+
```

Comment:
why you added this?! it'll impact all other projects! 
you can define something like this with class name not just li.

@ahmadmohammadirad2006 I expected to be more careful about approving these kinds of things, please be careful for shared directory

Reply:
@MichaelAndish I will pay more attention next time